### PR TITLE
ZTE-SH-CN-define-environment-variable-container translate

### PR DESCRIPTION
**Pull request by YuxiJin-tobeyjin**
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.8 Features: set Milestone to `1.8` and Base Branch to `release-1.8`
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>
> NOTE: Please check the “Allow edits from maintainers” box (see image below) to
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox
https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/tasks/inject-data-application/define-environment-variable-container.md

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5425)

### CODE REVIEW

File: cn/docs/tasks/inject-data-application/define-environment-variable-container.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        ...
+        DEMO_GREETING=Hello from the environment
+
+1. 通过键入`exit`退出命令终端。
+
+{% endcapture %}
+
+{% capture whatsnext %}
+
+* 获取更多资讯可参考 [environment variables](/docs/tasks/configure-pod-container/environment-variable-expose-pod-information/).
```
Comment (tengqm) [inline cn/docs/tasks/inject-data-application/define-environment-variable-container.md]: 有关环境变量的更多信息，请参阅[这里](/link)

File: cn/docs/tasks/inject-data-application/define-environment-variable-container.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+* 获取更多资讯可参考 [using secrets as environment variables](/docs/user-guide/secrets/#using-secrets-as-environment-variables).
```
Comment (tengqm) [inline cn/docs/tasks/inject-data-application/define-environment-variable-container.md]: 同上

**Comment by YuxiJin-tobeyjin** [comment]
@tengqm Thanks a lot,I will modify it.

File: cn/docs/tasks/inject-data-application/define-environment-variable-container.md [hunk 3]
```diff
@@ -61,8 +61,8 @@ title: 为容器设置环境变量
 
 {% capture whatsnext %}
 
-* 获取更多资讯可参考 [environment variables](/docs/tasks/configure-pod-container/environment-variable-expose-pod-information/).
-* 获取更多资讯可参考 [using secrets as environment variables](/docs/user-guide/secrets/#using-secrets-as-environment-variables).
+* 有关环境变量的更多信息，请参阅这里 [environment variables](/docs/tasks/configure-pod-container/environment-variable-expose-pod-information/).
```
Comment (tengqm) [inline cn/docs/tasks/inject-data-application/define-environment-variable-container.md]: sigh， I meant this:
```
* 有关环境变量的更多信息，请参阅[这里](/docs/tasks/configure-pod-container/environment-variable-expose-pod-information/)。
* 有关如何通过环境变量来使用Secret，请参阅[这里](/docs/user-guide/secrets/#using-secrets-as-environment-variables)。
* 关于[EnvVarSource](/docs/api-reference/{{page.version}}/#envvarsource-v1-core)资源的信息。
```

**Comment by YuxiJin-tobeyjin** [comment]
squash into one commit and resubmit

**Pull request closed without merging**